### DISCUSSION TITLE
Slow startup of Zettlr when Zettlr website was down. Is there any explanation for that correlation?

### DISCUSSION THREAD

**Discussion by fmartinp** [Zettlr/Zettlr#3332] [General]
Hello,

I have noticed that during the downtime of Zettlr website on 20/03/2022, Zettlr software was taking a significant more amount of time (several seconds) to load. When Zettlr website was up again, Zettlr's startup time when back to nominal.

The automatic update was disabled in my configuration, so I do not understand this correlation between the Zettlr website downtime and the slowness of the startup of the software if there is no other connection between Zettlr host and the software.

I am using the Zettlr 2.2.4 for Windows x64 (Zettlr-2.2.4-x64.exe)

Could anyone have an explanation for that correlation of the slow startup and the website downtime?

Thanks in advance.

**Comment by nathanlesage** [comment]
There are two instances where Zettlr tries to access the server: On update check, and to see if any translations are updated. The translation updates should already be detached from booting (i.e. it attempts all of that in the background), but maybe the update check is still blocking.

I'll have to double check that. Converting to issue …

**Reply by fmartinp** [reply]
Thanks for looking at the issue. It will be nice that the user can control (enable/disable) any connection to Zettlr server like the translations update too.

Thanks!
